src/args/arg.rs

```diff
@@ -4074,6 +4074,10 @@ impl<'a, 'b, 'z> From<&'z Arg<'a, 'b>> for Arg<'a, 'b> {
     fn from(a: &'z Arg<'a, 'b>) -> Self { a.clone() }
 }
 
+impl<'a, 'b> From<&'a str> for Arg<'a, 'b> {
```

Comment:
@kbknapp I think this is the only interesting part in the PR. The rest is just one `sed` command.